docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
 ---
-title: "User administration"
+title: "Users"
 weight: 1
-sidebarTitle: "Users"
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a role that controls their access and permission levels.
 ---
 
 {{< description >}}
 
-Any user added to a project or an environment type on Platform.sh will need to [register for an account](https://auth.api.platform.sh/register) before they can contribute. If you need to delete your account at any time you can [transfer ownership](/administration/users.md#transfer-ownership) of your existing projects and then open a [support ticket](/development/troubleshoot.md#deleting-your-platformsh-account) to request your account deletion.  
+If you are an organization or a Project admin wanting to add a user to a project or an environment, the user has to create an account before they can contribute to the project.
```

Comment:
"You" can be an org owner, but not an org. I think it's probably that org owners are automatically project admins, so I don't think both things are necessary.

Also, this comes before any introduction to the general topic, since that was removed from the description. So something there might help. Otherwise, it feels like a barrier is put in place before the end goal is even set, if that makes sense.

Reply:
Ok